src/buff-util.mjs

```diff
@@ -57,17 +63,22 @@ export function slice(buffer, start, end) {
 
 // NOTE: EXIF strings are ASCII encoded, but since ASCII is subset of UTF-8
 //       we can safely use it along with TextDecoder API.
-export function toString(buffer, start, end) {
+export function toString(buffer, start, end, clean = false) {
+	let result = null;
 	if (buffer instanceof DataView) {
 		if (hasBuffer) {
-			return Buffer.from(buffer.buffer)
+			result = Buffer.from(buffer.buffer)
 				.slice(start, end)
 				.toString('ascii', start, end)
 		} else {
 			var decoder = new TextDecoder('utf-8')
-			return decoder.decode(slice(buffer, start, end))
+			result = decoder.decode(slice(buffer, start, end))
 		}
 	} else {
-		return buffer.toString('ascii', start, end)
+		result = buffer.toString('ascii', start, end)
+	}
+	if (clean) {
```

Comment:
The changes in this function allow cleaning the string by removing everything after the first null character.